File: implement/lucky_straight.py
# 럭키 스트레이트

# 점수 n
n = input()

l1 = list(map(int, n[:len(n)//2]))
l2 = list(map(int, n[len(n)//2:]))

if sum(l1) == sum(l2):
   print("LUCKY")
else:
   print("READY")

File: java/Lucky_Straight.java
import java.util.Scanner;

public class Lucky_Straight {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);
		String s = sc.nextLine();
		char[] c = s.toCharArray();
		
		int sum = 0;
		for(int i=0; i<c.length; i++) {
			if(i< c.length/2) {
				sum+= c[i]-'0';
			} else {
				sum-= c[i]-'0';
			}
		}
		
		if(sum==0) {
			System.out.println("LUCKY");
		}else {
			System.out.println("READY");
		}
	}

}
